CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2021.6.999 (11 June 2021)
```

Comment:
Per @joyceerhl 's comment I agree that the num here doesn't really matter. Did the extra 9 just to distinguish it. Per the other point releases only pulled in the needed section.